client/SDL/sdl_freerdp.c

```diff
@@ -611,16 +612,25 @@ static BOOL sdl_create_windows(sdlContext* sdl)
 
 	rc = TRUE;
 fail:
+
 	if (!SetEvent(sdl->windows_created))
```

Comment:
hum strange we notify the event even in the failure case...

Reply:
we don´t want to lock up here, just return an error. (some thread is waiting on the other end)